math/polymake/Portfile

```diff
@@ -29,22 +29,31 @@ depends_lib-append      port:bliss \
                         port:gmp \
                         port:libxml2 \
                         port:libxslt \
+                        port:lrslib \
                         port:mpfr \
+                        port:normaliz \
                         port:p${perl5.major}-extutils-embed \
                         port:p${perl5.major}-json \
                         port:p${perl5.major}-svg \
                         port:p${perl5.major}-term-readkey \
                         port:p${perl5.major}-term-readline-gnu \
                         port:p${perl5.major}-xml-sax \
                         port:p${perl5.major}-xml-writer \
+                        port:permlib \
                         port:ppl \
                         port:readline \
                         port:scip \
                         port:singular \
-                        port:soplex
+                        port:soplex \
+                        port:sympol
 
 configure.perl          ${perl5.bin}
 configure.args          --with-boost=[boost::install_area] \
```

Comment:
Unless you want to replace any args added by portgroups, etc, use `configure.args-append`.